conventional_pre_commit/format.py

```diff
@@ -26,8 +27,20 @@ def r_types(types):
     return "|".join(types)
 
 
-def r_scope(optional=True):
+def _get_scope_pattern(scopes: Optional[List[str]] = None):
+    scopes_str = r_types(scopes)
+    escaped_delimiters = list(map(re.escape, [":", ","]))  # type: ignore
```

Comment:
Noting that this supports scopes such as `(api: client)` and `(api, client)` (from the tests, thank you!)

However it does not support delimiters such as `(api/client)` or `(api-client)`, as is the case with the baseline `r_scope()` .

Reply:
This should be resolved with commit f51f6621b3d4d0ca1cca5d226ed8d57bc2c032de